### PR TITLE
XW | small fix

### PR DESCRIPTION
**Pull request by tomasznapieralski**

### CODE REVIEW

**Comment by PiotrGackowski** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java [hunk 1]
```diff
@@ -127,37 +128,45 @@ public Navigation openRecentWikiActivity() {
...
       return false;
     }
   }
 
-  public boolean isFooterVisible() {
+  public boolean isLogoutLinkVisible() {
     try {
-      return homeOfFandomFooter.isDisplayed();
+      return logoutLink.isDisplayed();
     } catch (NoSuchElementException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java [hunk 2]
```diff
@@ -90,6 +79,29 @@ public void mercury_navigation_navigationOnNonEnglishWiki() {
...
+
+    topBar.openNavigation();
+    Assertion.assertTrue(navigation.isUserAvatarVisible());
+    Assertion.assertTrue(navigation.isUserProfileLinkVisible());
+    Assertion.assertTrue(navigation.isLogoutLinkVisible());
+  }
+
+  @Execute(asUser = User.ANONYMOUS)
+  @Test(groups = "mercury_navigation_navigationElementsAnonymousUser")
+  public void mercury_navigation_navigationElementsAnonymousUser() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/TopBarTests.java [hunk 3]
```diff
@@ -70,4 +70,28 @@ public void mecury_topbar_switchBetweenSearchAndNavigation() {
...
+  @Execute(onWikia = MercuryWikis.DE_WIKI)
+  @Test(groups = "mercury_topbar_fandomBarIsNotVisibleOnNonEnglishCommunity")
+  public void mercury_topbar_fandomBarIsNotVisibleOnNonEnglishCommunity() {
+    init();
+
+    Assertion.assertFalse(topBar.isFandomBarVisible());
+  }
+
+  @Test(groups = "mercury_topbar_wikiaLogoRedirectsToFandomPage")
+  public void mercury_topbar_wikiaLogoRedirectsToFandomPage() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/TopBarTests.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/TopBarTests.java [hunk 4]
```diff
@@ -70,4 +70,28 @@ public void mecury_topbar_switchBetweenSearchAndNavigation() {
     Assertion.assertTrue(topBar.isCloseIconVisible());
     Assertion.assertTrue(topBar.isSearchIconVisible());
   }
+
+  @Test(groups = "mercury_topbar_fandomBarIsVisibleOnEnglishCommunity")
+  public void mercury_topbar_fandomBarIsVisibleOnEnglishCommunity() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/TopBarTests.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/elements/mercury/components/TopBar.java [hunk 5]
```diff
@@ -115,4 +124,13 @@ public boolean isCloseIconVisible() {
       return false;
     }
   }
+
+  public boolean isFandomBarVisible() {
+    try {
+      return fandomBar.isDisplayed();
+    } catch (NoSuchElementException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/mercury/components/TopBar.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues:
* MAJOR 7 major

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues:
* MAJOR 7 major

Watch the comments in this conversation to review them.

**Pull request merged**